core/text/formatters.py

```diff
@@ -239,6 +239,7 @@ def de_camel(text: str) -> str:
     CodeFormatter("SNAKE_CASE", "_", lower, lower),
     CodeFormatter("DASH_SEPARATED", "-", lower, lower),
     CodeFormatter("DOT_SEPARATED", ".", lower, lower),
+    CodeFormatter("COMMA_SEPARATED", ", ", lower, lower),
```

Comment:
We already have a formatter with this name. See line 229 above. If there is something with the behavior of the old formatter you want to change please write a test for that.

Reply:
Oops, sorry for missing that